src/cosmos/SwapSkeleton.fixture.tsx

```diff
@@ -0,0 +1,25 @@
+import { darkTheme, defaultTheme, lightTheme, SwapWidgetSkeleton } from '@uniswap/widgets'
+import Row from 'components/Row'
+import { useEffect } from 'react'
+import { useValue } from 'react-cosmos/fixture'
+
+function Fixture() {
+  const [width] = useValue('width', { defaultValue: 360 })
+
+  const [theme, setTheme] = useValue('theme', { defaultValue: defaultTheme })
+  const [darkMode] = useValue('darkMode', { defaultValue: false })
+  useEffect(() => setTheme((theme) => ({ ...theme, ...(darkMode ? darkTheme : lightTheme) })), [darkMode, setTheme])
+  const widget = <SwapWidgetSkeleton theme={theme} width={width} />
+
+  // If framed in a different origin, only display the SwapWidget, without any chrome.
```

Comment:
This is unnecessary - it's only used for the Widget so it can be iframed in the docs.